functions/HttpTriggerAPIUsers/__init__.py

```diff
@@ -1,24 +1,52 @@
+import json
+import pypyodbc
 import logging
-
+import os
 import azure.functions as func
 
-
 def main(req: func.HttpRequest) -> func.HttpResponse:
-    logging.info('Python HTTP trigger function processed a request.')
+    logging.info('Python HTTP Database trigger function processed a request.')
+    
+    # Database credentials.
+    server = os.environ["ENV_DATABASE_SERVER"]
+    database = os.environ["ENV_DATABASE_NAME"]
+    username = os.environ["ENV_DATABASE_USERNAME"]
+    password = os.environ["ENV_DATABASE_PASSWORD"]
+    
+    # Define driver
+    driver = '{ODBC Driver 17 for SQL Server}'
+    
+    # Define the connection string
+    connection_string = "Driver={};Server={};Database={};Uid={};Pwd={};Encrypt=yes;TrustServerCertificate=yes;Connection Timeout=30;".format(driver, server, database, username, password)
+    
+    # Create a new connection
+    try:
+        with pypyodbc.connect(connection_string) as conn:
+            return select_users_query(conn)
+    except pypyodbc.DatabaseError as e:

```

Comment:
Nice! We might need to add additional error handling later  for the DB to get better responses for the user

Reply:
Thanks Jenny. I would add more error handling but I spent a lot of time trying to fix my issue with importing pyodbc and drivers.